isis/tests/cnetcheckFunctionalTests.cpp

```diff
@@ -0,0 +1,164 @@
+#include <iostream>
+#include <QTemporaryDir>
+
+#include "cnetcheck.h"
+
+#include "Cube.h"
+#include "CubeAttribute.h"
+#include "PixelType.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+#include "TestUtilities.h"
+#include "FileName.h"
+#include "ControlNet.h"
+#include "SerialNumber.h"
+#include "ControlMeasure.h"
+#include "ControlPoint.h"
+
+#include "Fixtures.h"
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/cnetcheck.xml").expanded();
+
+TEST_F(ThreeImageNetwork, FunctionalTestCnetcheckCamera) {
```

Comment:
probably worth updating these tests to have Functional in the name over app test.